django_select2/admin.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""
+Django-Select2 Tools for Django Admin.
+
+    Sample usage:
+        from django_select2.admin import Select2ModelAdmin
+        from django_select2.forms import Select2Widget
+
+
+        class MyModelAdmin(Select2ModelAdmin):
+            select2_fields = {
+                'my_choice_field': {
+                    'widget': Select2Widget,
+                },
+                'my_FK_field': {
+                    'widget_kwargs': {
+                        'search_fields': ['name__icontains'],
+                        'attrs': {
+                            'data-minimum-input-length': 2,
+                            'style': 'width:200px;',
+                        },
+                    },
+                },
+            }
```

Comment:
Since I accepted this correction, I had not commented on it. Maybe this way it looks like I am still in doubt about it. I prefer pushing all changes together but, if you say so, I will first push this smaller fixes first.